### DISCUSSION TITLE
Pagination and other issues

### DISCUSSION THREAD

**Discussion by zazou-d96** [lardbit/nefarious#219] [Ideas]
Hello. Still loving Nefarious but a few suggestions.

1. When paging through Wanted or Watched lists, clicking the back button, takes you all the way back to the first page. Same thing happens if an item is deleted. Not a major issue, but I'm sure it is one that can be easily fixed.
2.  Would it be possible for Nefarious to report that a dl has not found any sources in a configurable time period, (30,45,60days) and offer the delete/keep option?
3. Re managing large collections, one does occasionally repeat a download unnecessarily. It would be crazy if Nefarious could offer an option to scan through a text/csv file, before adding the download to the queue, and only raise an alert if it finds a similar file (title, year) and offer a cancel or continue option.

**Comment by lardbit** [comment]
Thanks for the suggestions.  

1. certainly doable
2. I'm not sure I see the benefit of this one.  You just want to remove it from your "watch" list if it fails to find a source within a month or two?  Rather than it just continue to look?
3. How do you accidentally repeat a download? nefarious can scan your entire library so it'll know exactly what you have.  See the main settings page for that option.  **Import Library**

**Reply by zazou-d96** [reply]
Hello,
Re the last point, many of us do not use Nefarious in the way you suggest. I have it on a 1tb laptop, downloads are scrapped with tinymediamanager and then moved to external storage, where depending on quality nad taste, it either goes into the collection or is deleted. So Nefarious idoes not access or manage my collection.

**Reply by lardbit** [reply]
Interesting.  You're right, I don't use nefarious like that but I do like the second idea of at least having nefarious communicate an item still hasn't found a source after a certain time period.  It supports push notifications already, so it could communicate that way.  I don't think I'll ever implement the third idea, though, but am open to contributions.
